src/coord/src/sink_connector.rs

```diff
@@ -118,34 +118,66 @@ async fn get_latest_ts(
     // Seek to end-1 offset
     let mut tps = TopicPartitionList::new();
     tps.add_partition(consistency_topic, partition);
-    tps.set_partition_offset(consistency_topic, partition, Offset::OffsetTail(1))?;
+    tps.set_partition_offset(consistency_topic, partition, Offset::Beginning)?;
```

Comment:
do we still need to manually set the offset to beginning or does that happen by default?

Reply:
I think it will work (today) but thought being explicit would be better here. WDYT?

Reply:
I think it will work (today) but thought being explicit would be better here. WDYT?

Reply:
thats fine i was just curious